internal/packages/assets.go

```diff
@@ -159,7 +162,7 @@ func loadElasticsearchAssets(pkgRootPath string) ([]Asset, error) {
 }
 
 func loadFileBasedAssets(kibanaAssetsFolderPath string, assetType AssetType) ([]Asset, error) {
-	assetsFolderPath := filepath.Join(kibanaAssetsFolderPath, string(assetType))
+	assetsFolderPath := filepath.Join(kibanaAssetsFolderPath, strings.ReplaceAll(string(assetType), "-", "_"))
```

Comment:
Folder in packages is names `security_rule`, but the type name is `security-rule`.

Reply:
Nit. Maybe add this as a comment to understand why we do the replace? Or maybe we could make `AssetType` more explicit, with a type that contains the asset type name and the folder name.